wavefront-adapter-for-istio/README.md

```diff
@@ -22,7 +42,7 @@ helm install wavefront-adapter-for-istio wavefront/wavefront-adapter-for-istio -
 
 ## Configuration
 
-The [values.yaml](./values.yaml) file contains information about all configuration
+The [values.yaml](https://raw.githubusercontent.com/wavefrontHQ/helm/master/wavefront-adapter-for-istio/values.yaml) file contains information about all configuration
```

Comment:
Any specific reason for this change? If we ever publish github releases  / tags etc, this will always point to master. Better to leave as is.

Reply:
The link is broken on helm hub page (https://hub.helm.sh/charts/wavefront/wavefront-adapter-for-istio) it is pointing to https://hub.helm.sh/values.yaml. So i changed to github link.